frontend/public/components/utils/rbac.tsx

```diff
@@ -90,7 +92,7 @@ type RequireCreatePermissionProps = {
 export const asAccessReview = (kindObj: K8sKind, obj: K8sResourceKind, verb: K8sVerb): AccessReviewResourceAttributes => ({
   group: kindObj.apiGroup,
   resource: kindObj.plural,
-  name: obj.metadata.name,
-  namespace: obj.metadata.namespace,
+  name: getName(obj),
+  namespace: getNamespace(obj),
```

Comment:
Should we be returning no access review in this case? Or is the intent to check if you can perform the action on all resources of this kind in the cluster?

Reply:
I'm concerned this is going to mask bugs where we're not passing the object when we mean to. Note that for normal users this will almost always disable the action (and it would be really easy to miss if testing as kubeadmin). If we want to check create, we should have a different utility imo. For namespaced resources, you would want to pass the create namespace as well.

Reply:
I see, we need another way for testing the permission.

I would still prefer to use this selector though and use some other way to test for nulls than crashing the UI.

Closing the PR

Reply:
We can add a null check, but we should be permissive by default if the object is not set. Maybe logging a warning is enough. But really this should never be null if it's a kebab action.

Reply:
ok, I added the null check.

it can happen quite easily to miss the null check when used like this: https://github.com/openshift/console/blob/e1df91c790eeb1b2ca3c7efc8511a4ca63058cd9/frontend/packages/metal3-plugin/src/components/host-menu-actions.tsx#L52

some cases similar to these could be easily missed in the future